src/types/state.js

```diff
@@ -51,7 +51,27 @@ export type RightClickedMarker = {|
   +markerIndex: MarkerIndex,
 |};
 
-export type ProfileViewState = {|
+/**
+ * Full profile view state
+ * They should not be used from the active tab view.
+ * NOTE: This state is empty for now, but will be used later, do not remove.
+ * globalTracks and localTracksByPid states will be here in the future.
+ */
+export type FullProfileViewState = {||};
+
+/**
+ * Active tab profile view state
+ * They should not be used from the full view.
+ */
+export type ActiveTabProfileViewState = {|
+  hiddenGlobalTracksGetter: () => Set<TrackIndex>,
+  hiddenLocalTracksByPidGetter: () => Map<Pid, Set<TrackIndex>>,
```

Comment:
I could now remove the wrapper functions and directly keep the Set and Map itself here. But they are actually being removed on the next PRs, so I wanted to just keep it as is for now.